<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>129</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Server IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>171</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Server Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>171</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="server_ip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>261</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="server_port">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>261</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>261</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>170</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
